src/main/java/Deadline.java

```diff
@@ -0,0 +1,14 @@
+public class Deadline extends Task {
+
+    protected String by;
+
+    public Deadline(String description, String by) {
+        super(description);
+        this.by = by;
+    }
+
+    @Override
+    public String toString() {
```

Comment:
I like how you have overriden the `toString()` method to make it specific to the deadline class. Remember also to include in your javadoc for this method to provide a justification as to why you have overriden the `toString()` method 👍